kong/tools/table.lua

```diff
@@ -111,25 +111,13 @@ do
 
     return is_array_strict(t)
   end
-end
 
 
---- Checks if a table is an array and not an associative array.
--- *** NOTE *** string-keys containing integers are considered valid array entries!
--- @param t The table to check
--- @return Returns `true` if the table is an array, `false` otherwise
-function _M.is_lapis_array(t)
-  if type(t) ~= "table" then
-    return false
-  end
-  local i = 0
-  for _ in pairs(t) do
-    i = i + 1
-    if t[i] == nil and t[tostring(i)] == nil then
-      return false
-    end
-  end
-  return true
+  --- Checks if a table is an array and not an associative array.
+  -- *** NOTE *** string-keys containing integers are considered valid array entries!
+  -- @param t The table to check
+  -- @return Returns `true` if the table is an array, `false` otherwise
+  _M.is_lapis_array = is_array_lapis
```

Comment:
Please unify the name rather than presenting the same function under two names.

Reply:
`is_array_lapis()` is a local function which is called by `is_array()`, and there similar functions like `is_array_fast` `is_array_strict`.

I did not change the function name just because I don't want to break the code.